ch10.md

```diff
@@ -107,20 +106,16 @@ Again, because we're using partial application to achieve this result, we must e
 Let's look at another example.
 
 ```js
-// Helpers:
-// ==============
-//  $ :: String -> IO DOM
-var $ = function(selector) {
-  return new IO(function() { return document.querySelector(selector)});
-}
+//-- Helpers ----------------------------------------------------------
+//    $ :: String -> IO DOM
+const $ = selector => new IO(_ => document.querySelector(selector))
 
-//  getVal :: String -> IO String
-var getVal = compose(map(_.prop('value')), $);
```

Comment:
My idea when I started with this is splicing in js-snippets from a bigger complete file that also imports the support functions.

Again going for the runnable code.

I just want to get into a better iterative process since this PR is all ready huge. It would be nice to get to a state that can be merged and then work on making everything more interactive.